artie/README.md

```diff
@@ -0,0 +1,52 @@
+# Artie Integration
+
+## Overview
+
+[Artie][1] is a real-time database replication product for reliably and consistently syncing data from source databases to data warehouses.
```

Comment:
Can we do [Artie](https://www.artie.com)?

Reply:
Instead of what?

Reply:
Like this instead of the footnotes:

```md
Can we do [Artie](https://www.artie.com)? 
```

Reply:
Oh it's the same in practice, but datadog's style guidelines want links to be formatted like this.

Reply:
Here's how it looks: https://github.com/artie-labs/integrations-extras/blob/bf1b04b1908d3497f6a7133b11506bbcd0813cea/artie/README.md

Reply:
Oh okay cool